schema/spec.md

```diff
@@ -100,83 +290,33 @@ this form:
 }
 ```
 
-## Notations and Terminology
+## 4. Schema Registry
 
-### Notational Conventions
+The Schema Registry is a metadata store for organizing schemas and schema
+Versions of any kind; it is a document store.
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
-interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+Implementations of this specification MAY include additional extension
+attributes, including the `*` attribute of type `any`.
 
-For clarity, OPTIONAL attributes (specification-defined and extensions) are
-OPTIONAL for clients to use, but the servers' responsibility will vary.
-Server-unknown extension attributes MUST be silently stored in the backing
-datastore. Specification-defined, and server-known extension, attributes MUST
-generate an error if the corresponding feature is not supported or enabled.
-However, as with all attributes, if accepting the attribute would result in a
-bad state (such as exceeding a size limit, or results in a security issue),
-then the server MAY choose to reject the request.
+Since the Schema Registry is an application of the xRegistry specification, all
+attributes for Groups, Resources, and Resource Version objects are inherited
+from there.
 
-In the pseudo JSON format snippets `?` means the preceding attribute is
-OPTIONAL, `*` means the preceding attribute MAY appear zero or more times,
-and `+` means the preceding attribute MUST appear at least once. The presence
-of the `#` character means the remaining portion of the line is a comment.
-Whitespace characters in the JSON snippets are used for readability and are
-not normative.
+### 4.1. Schema Groups
 
-### Terminology
+The Group (`<GROUP>`) name for the Schema Registry is `schemagroups`. The
```

Comment:
this is switched up: GROUP->schemagroup, GROUPS->schemagroups.
You may want to explicitly mention one "singular" vs "plural", not just the plural name itself